npm does not work when using --prebuilt

``` console
[kyle:~/testing_prebuilt] [testing_prebuilt] $ npm -v

module.js:340
    throw err;
          ^
Error: Cannot find module 'npmlog'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at /Users/kyle/c9/testing_prebuilt/bin/npm:19:11
    at Object.<anonymous> (/Users/kyle/c9/testing_prebuilt/bin/npm:87:3)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
```

--- a/nodeenv.py
+++ b/nodeenv.py
@@ -381,7 +381,7 @@ def copy_node_from_prebuilt(env_dir, src_dir):
     Copy prebuilt binaries into environment
     """
     logger.info('.', extra=dict(continued=True))
-    callit(['cp', '-r', src_dir + '/node-v*/*', env_dir], True, env_dir)
+    callit(['cp', '-a', src_dir + '/node-v*/*', env_dir], True, env_dir)
     logger.info('.', extra=dict(continued=True))
 
 

